src/Ionide.ProjInfo/Library.fs

```diff
@@ -764,10 +767,30 @@ type WorkspaceLoaderViaProjectGraph private (toolsPath, ?globalProperties: (stri
             pg
 
     let loadProjects (projects: ProjectGraph, customProperties: string list, binaryLogs: BinaryLogGeneration) =
+        let handleError (e: exn) =
+            let msg = e.Message
+            printfn "error -> %A" e
+            logger.error (Log.setMessage "Failed loading" >> Log.addExn e)
+
+            projects.ProjectNodesTopologicallySorted
```

Comment:
Since you call out `ProjectNodesTopologicallySorted` below, does this usage of it need to change as well?

Reply:
Done